### DISCUSSION TITLE
عدم اجرای تابع _onlinePayment.Verify

### DISCUSSION THREAD

**Discussion by shafie81** [Sina-Soltani/Parbad#239] [Q&A]
در زمان برگشت از بانک تابع verify پاسخ نمیده و پروژه در حالت انتظار می ماند

**Comment by Sina-Soltani** [comment]
اطلاعات زیر رو لطفا پاسخ بدید

نوع پروژه؟
نوع سرور؟
شماره نسخه Parbad
